src/main/kotlin/no/nav/familie/ef/sak/økonomi/dto/TilkjentYtelseRestDTO.kt

```diff
@@ -0,0 +1,81 @@
+package no.nav.familie.ef.sak.økonomi.dto
+
+import java.lang.IllegalStateException
+import java.time.LocalDate
+
+data class TilkjentYtelseRestDTO(
+    val søker: String,
+    val saksnummer: String,
+    val andelerTilkjentYtelse: List<AndelTilkjentYtelseRestDTO>
+) {
+
+    fun valider() {
+        if(andelerTilkjentYtelse.size==0) {
```

Comment:
Dersom det viser seg at en søknad tilsier at vedkommende ikke kvalifiserer for ytelser - hvordan skal dette løses?

Reply:
Da bør det ikke finnes en tilkjent ytelse. Eller at den tilkjent ytelse settes til avsluttet. 

Men her er det usikkerhet rundt modellen. BA opererer med Fagsak, Behandling, BehandlingResultat, Vedtak og TilkjentYtelse, som sammen kontrollerer tilstanden til søkeren. Jeg må innrømme at jeg har full kontroll på kravene der, og foreløpig prøvd å ivareta alt i TilkjentYtelse, som sikkert må splittes opp etter hvert

Reply:
Aha, da skjønner jeg.